carcara/src/checker/compression/mod.rs

```diff
@@ -0,0 +1,407 @@
+#![allow(unused_must_use)]
```

Comment:
The formatting of this file is not consistent with the rest of the project. Maybe try to run `rustfmt` on it.